package.json

```diff
@@ -11,29 +11,30 @@
     "@babel/plugin-proposal-decorators": "^7.10.5",
     "@babel/preset-env": "^7.10.4",
     "@babel/preset-react": "^7.10.4",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.4.2",
     "babel-eslint": "^10.1.0",
     "babel-loader": "^8.1.0",
     "concurrently": "^5.2.0",
     "css-loader": "^3.6.0",
     "eslint": "^7.5.0",
     "eslint-config-airbnb": "^18.2.0",
+    "eslint-import-resolver-webpack": "^0.13.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-jsx-a11y": "^6.3.1",
     "eslint-plugin-react": "^7.20.3",
     "eslint-plugin-react-hooks": "^4.0.8",
     "file-loader": "^6.0.0",
     "html-webpack-plugin": "^4.3.0",
     "nodemon": "^2.0.4",
+    "react-refresh": "^0.8.3",
     "style-loader": "^1.2.1",
     "url-loader": "^4.1.0",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.12",
-    "webpack-merge": "^5.0.9",
-    "@pmmmwh/react-refresh-webpack-plugin": "^0.4.2",
```

Comment:
It looks like removing this breaks hot reloading.

Reply:
hmm, I didn't remove it, it's just moved up because npm installing auto-generates this list.

Reply:
Yeah just noticed that too